ccp/src/main/res/raw/ccp_marathi.xml

```diff
@@ -113,12 +113,12 @@
             name_code="bf"
             phone_code="226" />
         <country
-            name="बल्गेरिया"
+            name="बल्जेरिया"
             english_name="Bulgaria"
             name_code="bg"
             phone_code="359" />
         <country
-            name="बहरीन"
+            name="बऱ्हेन"

```

Comment:
I believe this was already correct translation. Isn't it?
![image](https://user-images.githubusercontent.com/4918760/100390463-ae1fb400-2fed-11eb-887d-3a899c934463.png)